cfgov/v1/models/molecules.py

```diff
@@ -75,3 +117,14 @@ class TextIntroduction(blocks.StructBlock):
     class Meta:
         icon = 'title'
         template = 'v1/demo/molecules/text_introduction.html'
+
+
+class CallToAction(blocks.StructBlock):
+    slug = blocks.CharBlock(blank=False)
```

Comment:
Yes, this was a typo. Switching back and forth between these arguments is confusing sometimes :confounded: